linux_os/guide/system/permissions/restrictions/coredumps/coredump_disable_storage/bash/shared.sh

```diff
@@ -6,4 +6,4 @@
 . /usr/share/scap-security-guide/remediation_functions
 include_lineinfile
```

Comment:
include is not needed anymore with macro

Reply:
thanks